utils/src/server/rpc_module.rs

```diff
@@ -180,6 +180,18 @@ impl Methods {
 		}
 	}
 
+	/// Helper alternative to `execute`, useful for writing unit tests without having to spin
+	/// a server up.
+	///
+	/// Converts the params to an array for you if it's not already serialized to a sequence.
+	pub async fn call_with<T: Serialize>(&self, method: &str, params: &T) -> Option<String> {
```

Comment:
I think this (and `call`) should be `#[cfg(test)]`, wdyt?

Reply:
#[cfg(test)] doesn't work to shared with other crates IIRC, you need to hide behind some feature such `test-helper` or something like that, but was an anti-pattern with the `feature leakage`

Reply:
Ofc, I'm dumb. That's annoying.

Reply:
I can rename the function to `test_call_rpc_method` if you think it's more clear but I prefer not to feature-gate it.

Reply:
For the subscription-side of this I called it [`test_subscription()`](https://github.com/paritytech/jsonrpsee/pull/459/files#diff-7cf339924bf9f55b7b2a32c481d69d8e70114e91a0c074b7228bf7f80f964f8aR233). Not a very inspired name, but it gets the point across.

(I need to remove the `#[cfg(test)]` there though.)

Reply:
yepp, I think it's better here with test_*